[no-jira]: Adding test pre-reqs to ensure right environment for running (#66)
Skyscanner/full-stack-recruitment-test: README.md

----

## Prerequisites/Environment Setup

### Installing Node

The full stack test is developed using Node, using the following versions:

  **Node:** `LTS/Erbium (^12.13.0)`

  **npm:** `^6.12.0`

> If you use the latest versions of node and npm its recommended to use [nvm](https://github.com/nvm-sh/nvm) or [nave](https://github.com/isaacs/nave) to manage your Node environment on **Unix** systems. If you use **Windows** then its recommended to use [nvm-windows](https://github.com/coreybutler/nvm-windows) to manage your Node environment.

Reviewer: ```suggestion
We recommend using [nvm](https://github.com/nvm-sh/nvm) or [nave](https://github.com/isaacs/nave) to manage your Node environment on **Unix/macOS** systems. If you use **Windows** then we recommend using [nvm-windows](https://github.com/coreybutler/nvm-windows).
```

To install the correct npm version, use `npm install --global npm@^6.12.0`.

> To install the correct node version, use `nvm install lts/erbium`.

Reviewer 2: If you just do `nvm install` it'll install (and use) the version from `.nvmrc`

> To install the correct npm version, use `npm install --global npm@^6.12.0`.

Reviewer: This is done as part of nvm, so I think it confuses things to have this step

Author: Ah yes so actually it might be worth saying: `nvm install lts/erbium`? As just using `nvm use` will throw an error saying the version is not installed use `nvm install lts/erbium` to install

* `npm start` - This will start the application for development
* `npm run build` - Will create a production optimised build
* `npm test` - Will run the front end tests

> * `npm test:windows` - Will run the front end tests on Windows.

Reviewer: This is gone now
